site/lists/calendar.scroll

```diff
@@ -15,6 +15,14 @@ pipeTable
  Strange Loop 2022|StangeLoop|September|22|2022|09/04/22|Union Station, St. Louis, Missouri|USA|https://www.thestrangeloop.com/|strangeloop_stl
  RacketCon 2022|RacketCon|October|28|2022|10/30/22|Providence, Rhode Island|USA|https://con.racket-lang.org/|
  Systems, Programming, Languages and Applications: Software for Humanity 2021|SPLASH|December|5|2022|12/10/22|Milford Sound|New Zealand|https://2022.splashcon.org/|splashcon
+ Blockchain Over Coffee|BOC|October|6|2022|6/10/2022|San Francisco|USA|https://us02web.zoom.us/webinar/register/WN_Ie5qjPDvQEKQIv0psGqUJw|
+ International Conference on Software Programming Process and Web Development|ICSPP|November|3|2022|4/11/2022|Cape Town|South Africa|https://waset.org/software-programming-process-and-web-development-conference-in-november-2022-in-cape-town
+ Worldwide Software Architecture Summit'22|WSAS|November|15|2022|16/11/2022|Wilmington|USA|https://events.geekle.us/wsas4/?utm_source=google&utm_medium=cp%D1%81&utm_campaign=arc&utm_term=software%20development%20conferences&gclid=Cj0KCQjw1vSZBhDuARIsAKZlijRuK5JT6-HD9dG9Af6bZHFHUSQrOmmhQ-7nXKbsJSGMvDjO3u3tDPYaAhwMEALw_wcB#
```

Comment:
you can drop the query string from this url

Reply:
okay let me fix that